contracts/eden/src/actions/genesis.cpp

```diff
@@ -16,22 +35,24 @@ namespace eden
 
       auto inviter = get_self();
 
-      for (int i = 0; i < genesis_pool.size(); i++)
+      for (int i = 0; i < initial_members.size(); i++)
       {
          auto induction_id = i + 1;
-         const auto& invitee = genesis_pool[i];
-         auto total_endorsements = genesis_pool.size() - 1;
+         const auto& invitee = initial_members[i];
+         auto total_endorsements = initial_members.size() - 1;
 
          members.create(invitee);
-         inductions.create_induction(induction_id, get_self(), genesis_pool[i], total_endorsements);
+         inductions.create_induction(induction_id, get_self(), initial_members[i], total_endorsements);
 
-         for (const auto& endorser : genesis_pool)
+         for (const auto& endorser : initial_members)
          {
             if (endorser != invitee)
             {
                inductions.create_endorsement(inviter, invitee, endorser, induction_id);
             }
          }
+
+         inductions.update_video(inductions.get_induction(induction_id), genesis_video);
```

Comment:
I think it would be optimal if we add a new video string parameter (default to "") to `inductions.create_induction` method, then you can do it directly on emplace.

Reply:
I actually was getting trx timeout with 3 members on genesis... not sure if it's because its always resetting the endorsements... (pretty small actually, 3*3)... I did this adjust in my branch and it passes (even with 20 members): https://github.com/eoscommunity/Eden/pull/57/files